src/sdk/data/data.module.ts

```diff
@@ -249,4 +249,65 @@ export class DataModule {
       throw new Error(error.message || 'Failed to fetch exchange rates');
     }
   }
+
+  async getConnextSupportedAssets(chainId?: number): Promise<ConnextToken[]> {
+    try {
+      const queryParams = {
+        'api-key': this.currentApi,
+        chainId,
+      };
+
+      const result = await this.apiService.makeRequest(API_ENDPOINTS.GET_CONNEXT_SUPPORTED_ASSETS, MethodTypes.GET, queryParams);
+
+      return result ? result.tokens : [];
+    } catch (error) {
+      throw new Error(error.message || 'Failed to get connext supported assets');
+    }
+  }
+
+  async getConnextQuotes(
```

Comment:
Again, we ought to have a common endpoint called `getQuotes` with optional provider (set default as Connext).

Reply:
I have made this change.